pkg/loadbalance/weighted_round_robin.go

```diff
@@ -154,3 +156,12 @@ func (rp *RoundRobinPicker) Next(ctx context.Context, request interface{}) (ins
 	ins = rp.instances[idx]
 	return ins
 }
+
+func gcdInt(a, b int) int {
+	for b != 0 {
+		r := a % b
```

Comment:
```suggestion
		a, b = b, a%b
```
reduce variable declarations